### ISSUE TITLE
Overfitting experiments

### ISSUE THREAD

**Issue by Bochicchio3** [OpenDriveLab/OpenLane-V2#66]
I have a question regarding the resnet18 backbone.
Do you know if it works or was it ever successfully tested?  Since it's the only one that trains as is on V100 / A5000 gpus I have tried to overfit it to one sample, but it doesn't really work. I have tried many things, different learning rates, different learning rate schedulers, frozen training, predicting only lc and more. I tried for lots of epochs as well ( ~2000). 

Same overfitting experiment works on the internimage backbone ( starting from the provided checkpoint, at 1/4th resolution). Still have to test if the same training works from scratch. 

I am planning to test Resnet50 backbone at 1/4th resolution and update this post next, but I had hoped for the overfitting experiment on the small backbone to converge as well.

**Comment by faikit** [comment]
Hi, I think it would be better to adapt the r50 backbone in the [baseline_large.py](https://github.com/OpenDriveLab/OpenLane-V2/blob/master/plugin/models/configs/baseline_large.py) config to an r18 one, as the network designs are different in the two configs.
